<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">

QLabel {
	font: 300 60px &quot;Ubuntu&quot;;
	color: rgb(75, 0, 61);
	background: rgba(0, 0, 0, 0.0);
}

QPushButton {
	background: rgba(212, 83, 150, 0.3);
	border: 2px rgb(112, 21, 68);
	border-radius: 30px;
}


QPushButton:hover {
	background: rgba(245, 193, 220, 0.3);
}


QPushButton:pressed {
	background: rgba(152, 44, 101, 0.5)
}
</string>
  </property>
  <widget class="QPushButton" name="infoBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>960</y>
     <width>42</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(115, 187, 141);
	font: 35px;
	border: 10px white;
	border-radius: 20px;
	color: white;
}

QPushButton:hover {
    background: rgba(115, 187, 141, 0.5);
}


QPushButton:pressed {
    background: rgb(77, 141, 100);
}

</string>
   </property>
   <property name="text">
    <string>  i  </string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>960</y>
     <width>42</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(208, 178, 203);
	font: 35px;
	border: 10px white;
	border-radius: 20px;
	color: white;
}

QPushButton:hover {
    background: rgba(208, 178, 203, 0.5);
}


QPushButton:pressed {
    background: rgb(183, 130, 174);
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="preferences-desktop"/>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>67</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>C |</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>870</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(75, 0, 61);
	font: 300 70PX &quot;URW Bookman&quot;;
	background: rgba(0, 0, 0, 0)
}


QPushButton:hover {
	font: 80PX;
	color: rgba(75, 0, 61, 0.5);
}


QPushButton:pressed {
	font: 600 65PX &quot;URW Bookman&quot;;
	color: rgb(72, 36, 70);
}
</string>
   </property>
   <property name="text">
    <string>НАЧАТЬ</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>40</y>
     <width>541</width>
     <height>991</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background: rgba(0,0,0,0.);
	border: 1px black;
}

QPushButton {
	font: 30px;
	color: rgb(236, 221, 235);
	background: rgba(212, 83, 150, 0.3);
	border: 20px rgb(112, 21, 68);
	border-radius: 30px;
}


QPushButton:hover {
	background: rgba(245, 193, 220, 0.3);
}


QPushButton:pressed {
	background: rgba(152, 44, 101, 0.5)
}
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>527</width>
      <height>1548</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_14">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-6.png)</string>
       </property>
       <property name="text">
        <string>3260</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_13">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-5.png)</string>
       </property>
       <property name="text">
        <string>1575</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-1.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-2.png);
</string>
       </property>
       <property name="text">
        <string>2228</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-4.png)</string>
       </property>
       <property name="text">
        <string>1485</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="skin1Btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/german.png)</string>
       </property>
       <property name="text">
        <string>5555</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="skin3Btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/tanya.png)</string>
       </property>
       <property name="text">
        <string>5555</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="skin2Btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/katya.png);
color: black
</string>
       </property>
       <property name="text">
        <string>5555</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_12">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-3.png);</string>
       </property>
       <property name="text">
        <string>640</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="skin4Btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/nikita.png);</string>
       </property>
       <property name="text">
        <string>5555</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_16">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-7.png)</string>
       </property>
       <property name="text">
        <string>950</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_15">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/skins/resourses/images/skins/skin-8.png);
color: black</string>
       </property>
       <property name="text">
        <string>1350</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="skinImg">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>610</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>970</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px black;
border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
